contracts/amm/MinterAmm.sol

```diff
@@ -1070,4 +1070,16 @@ contract MinterAmm is
             );
         return (pricesStdVega.price, pricesStdVega.stdVega);
     }
+
+    function getUnderlyingToken() external view override returns (IERC20) {
+        return underlyingToken;
```

Comment:
Do we need functions for these getters or can we just query underlyingToken on the amm?  something like amm. underlyingToken since its public?

Reply:
Yeah this was ugly... the inheritance tree screwed this up since the interface that others reference don't know anything about the public variables in the storage contract.  Keep this comment open and we can investigate moving around the inheritance tree without affecting memory layout is possible.

Reply:
Not sure I fully understand. I'm already accessing this directly for example https://github.com/sirenmarkets/core/blob/4c33b0f164e72a2b1c0f134fafedda37804bf5a6/contracts/amm/AmmDataProvider.sol#L559